src/bootloader/bootloader.c

```diff
@@ -160,6 +160,26 @@ typedef union {
 
 static uint8_t _loading_ready = 0;
 static uint8_t _firmware_num_chunks = 0;
+// Indicates whether the whole app flash contains only 0xFF.
+// This controls bootloader text messages on the screen.
+// The value is computed at bootloader enter and at OP_ERASE behind a condition.
+static uint8_t _is_app_flash_empty = 0;
```

Comment:
we usually use `bool` for bools :P

Reply:
that's not what I'm seeing in the code:

- `_loading_read` is treated as bool
- some vars use `sectrue_u8` and `secfalse_u8`
- others do use false

wanted to be consistent with the other static vars.

Reply:
Great you found the only other example :D ok fine - but consider changing both to `bool` :)

Reply:
:) ok, will send you a separate PR

Reply:
https://github.com/digitalbitbox/bitbox02-firmware/pull/760